cassandra/scripts/start-cassandra.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+SCRIPT_DIR="$(dirname "$0")"
+
+kubectl label nodes 127.0.0.1 name=cassandra
+
+kubectl create -f $SCRIPT_DIR/../k8s/peer-service.yaml
+kubectl create -f $SCRIPT_DIR/../k8s/service.yaml
+kubectl create -f $SCRIPT_DIR/../k8s/daemonset.yaml --validate=false
```

Comment:
Validation not pass, k8s docs says it's ok. https://github.com/kubernetes/kubernetes/tree/master/examples/cassandra

This scripts will be deleted after implementing yagoda/issues/23